### ISSUE TITLE
Terminal class fails to determine witdth in NeoVim

### ISSUE THREAD

**Issue by weilbith** [greenbone/autohooks#46]
I just introduced `autohooks` into one of my project. :tada: 
Unfortunately get the error listed below, when I do my commit from within _NeoVim_ with the `fugitive` plugin for _Git_. Apparently does the printing to the terminal not work correctly here. Unfortunately does this block the commit for me.

```sh
autohooks => pre-commit
Running autohooks.plugins.black
Traceback (most recent call last):
  File "/home/thore/.cache/pypoetry/virtualenvs/gandyns-py3.8/lib/python3.8/site-packages/autohooks/precommit/run.py", line 122, in run
    retval = plugin.precommit(config=config.get_config())
  File "/home/thore/.cache/pypoetry/virtualenvs/gandyns-py3.8/lib/python3.8/site-packages/autohooks/plugins/black/black.py", line 82, in preco
mmit
    ok('No staged files for black available.')
  File "/home/thore/.cache/pypoetry/virtualenvs/gandyns-py3.8/lib/python3.8/site-packages/autohooks/api/__init__.py", line 35, in ok
    __term.ok(message)
  File "/home/thore/.cache/pypoetry/virtualenvs/gandyns-py3.8/lib/python3.8/site-packages/autohooks/terminal.py", line 65, in ok
    self._print_end(message, 'ok', self._term.green)
  File "/home/thore/.cache/pypoetry/virtualenvs/gandyns-py3.8/lib/python3.8/site-packages/autohooks/terminal.py", line 37, in _print_end
    width = self._term.width - 1
TypeError: unsupported operand type(s) for -: 'NoneType' and 'int'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ".git/hooks/pre-commit", line 8, in <module>
    sys.exit(run())
  File "/home/thore/.cache/pypoetry/virtualenvs/gandyns-py3.8/lib/python3.8/site-packages/autohooks/precommit/run.py", line 140, in run
    term.error(
  File "/home/thore/.cache/pypoetry/virtualenvs/gandyns-py3.8/lib/python3.8/site-packages/autohooks/terminal.py", line 71, in error
    self._print_end(message, 'error', self._term.red)
  File "/home/thore/.cache/pypoetry/virtualenvs/gandyns-py3.8/lib/python3.8/site-packages/autohooks/terminal.py", line 37, in _print_end
    width = self._term.width - 1
TypeError: unsupported operand type(s) for -: 'NoneType' and 'int'
```

**Comment by bjoernricks** [comment]
Hi thanks for choosing autohooks 😊

Would it be possible to try the master branch? We have several issues with the used terminal library blessings and already switched to a different solution in master. This may also have fixed your issue.

**Comment by weilbith** [comment]
Hmm I fail to do so. This command fails always:

```sh
$ poetry --version
Poetry 0.12.17
$ poetry add --dev --git https://github.com/greenbone/autohooks.git

[MissingArguments]
Not enough arguments

add [-D|--dev] [--git GIT] [--path PATH] [-E|--extras EXTRAS] [--optional] [--python PYTHON] [--platform PLATFORM] [--allow-prereleases] [--dry-run] [--] <name> (<name>)...
```

Will it take long until you release a new version from master?

**Comment by bjoernricks** [comment]
With poetry 1.0 it should be
```
poetry add --dev  https://github.com/greenbone/autohooks.git
```
Not sure but maybe you have to use
```
poetry add --dev --git https://github.com/greenbone/autohooks.git autohooks
```
with your version

**Comment by bjoernricks** [comment]
Should be fixed with autohooks 2.1.0.
